en/concepts/scene/scene-editing.md

```diff
@@ -0,0 +1,95 @@
+# Build a Scene Image Using the Scene Panel
+
+This document will introduce the use of the [Scene](../../editor/scene/index.md) panel to create and edit scene images.
```

Comment:
......will introduce the workflow and tips of using [Scene]() panel to create and editing scene contents.